src/ontodia/data/rdf/rdfCompositeParser.ts

```diff
@@ -0,0 +1,41 @@
+import {Dictionary} from '../model';
+
+export class RDFCompositeParser {
+    constructor(public parserMap: Dictionary<any>) { }
+
+    parse(body: string, mimeType?: string): Promise<any> {
+        if (mimeType) {
+            if (mimeType === 'application/rdf+xml') {
+                body = body.replace(/Collection/ig, 'Collection1');
```

Comment:
Why do we need to replace `Collection` with `Collection1`? Maybe you should add the reasoning as a comment here.